ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewGroupManager.java

```diff
@@ -48,6 +48,10 @@ public void removeViewAt(T parent, int index) {
     parent.removeViewAt(index);
   }
 
+  public void removeView(T parent, View view) {
+    parent.removeView(view);
```

Comment:
Unfortunately, adding this now isn't safe for all the implementations of this class that have already overridden the other addView, removeViewAt, etc. calls. Instead, I think we need to this in NativeViewHiearchyManager:

```
for (int i = 0; i < viewManager.getChildrenCount(viewToManage); i++) {
  if (viewManager.getChildAt(viewToManage, i) == viewToDelete) {
    viewManager.removeViewAt(viewToManager, i);
    break;
  }
}
```